posix/subsystem/src/file.cpp

```
@@ -266,6 +283,12 @@ async::result<frg::expected<Error, size_t>> File::writeAll(Process *, const void
 	throw std::runtime_error("posix: Object has no File::writeAll()");
 }
 
+async::result<frg::expected<Error, size_t>> File::pwrite(Process *, int64_t, const void *, size_t) {
+	std::cout << "posix \e[1;34m" << structName()
+			<< "\e[0m: Object does not implement pwrite()" << std::endl;
+	throw std::runtime_error("posix: Object has no File::pwrite()");
+}
```

Let's return an error instead of dying here. I guess the default error should be `ESPIPE`?

---

Done as requested.